docs/plugins/google-cloud.md

```diff
@@ -47,14 +47,49 @@ export default configureGenkit({
 
 When running in production, your telemetry gets automatically exported.
 
-The plugin requires the Google Cloud project ID and your Google Cloud project credentials. If you're running your flow from a Google Cloud environment (Cloud Functions, Cloud Run, etc), the project ID and credentials are set automatically. Running in other environments requires setting the `GCLOUD_PROJECT` environment variable to your Google Cloud project and authenticating using the `gcloud` tool:
+### Authentication
+
+The plugin requires the Google Cloud project ID and your Google Cloud project credentials. If you're running your flow from a Google Cloud environment (Cloud Functions, Cloud Run, etc), the project ID and credentials are set automatically. 
+
+#### Application Default Credentials
+
+Running in other environments requires setting the `GCLOUD_PROJECT` environment variable to your Google Cloud project and authenticating using the `gcloud` tool:
 
 ```posix-terminal
 gcloud auth application-default login
 ```
 
 For more information, see the [Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc) docs.
 
+#### Service Account Credentials
+
+If you are using a service account and running outside of a Google Cloud environment, you can set your credentials as an environment variable. Follow instructions here to [set up your Google Cloud Service Account Key](https://cloud.google.com/iam/docs/keys-create-delete#creating).
+
+Once you have downloaded the key file, you can specify the credentials in two ways a file location using the `GOOGLE_APPLICATION_CREDENTIALS` environment variable or directly copy the contents of the json file to the environment variable `GCLOUD_SERVICE_ACCOUNT`.
```

Comment:
Not a fan of `GCLOUD_SERVICE_ACCOUNT` env var name. It makes me thing it should be just the service account name, not the credentials JSON blob.

How about  `GOOGLE_SERVICE_ACCOUNT_CREDS_JSON` or at least `GOOGLE_SERVICE_ACCOUNT_CREDS`?

Reply:
yeah, I also didn't love the name, but also didn't love how long it can quickly get. If you're ok with `GOOGLE_SERVICE_ACCOUNT_CREDS`, I'll do that for brevity.